front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -64,17 +68,32 @@ const ScenarioContent = ({
   const [ngeIsLoading, setNGEIsLoading] = useState(true);
 
   const dtoImport = useCallback(async () => {
-    const timetablePromise = dispatch(
+    const timetableTrainSchedulesPromise = dispatch(
```

Comment:
Timetable is obivous here 
```suggestion
    const trainSchedulesPromise = dispatch(
```

Reply:
Done in "rename variables"